app/services/import/breweries.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
+
+module Import
+  class Breweries
+
+    attr_reader :log
+
+    def initialize
+      @log = ActiveSupport::Logger.new('log/import_breweries.log')
+      @dry_run = ENV['DRY_RUN'].present? ? ENV['DRY_RUN'].casecmp('true').zero? : false
+      @counter = { added: 0, failed: 0, skipped: 0, total: 0 }
+      @path_to_json = 'https://raw.githubusercontent.com/openbrewerydb/openbrewerydb/master/breweries.json'
+      @path_to_sql = 'https://raw.githubusercontent.com/openbrewerydb/openbrewerydb/master/breweries.sql'
+    end
+
+    def self.perform
+      new.perform
+    end
+
+    def perform
+      start_time = Time.now
+      puts "\nTask started at #{start_time}"
+      @log.info "Task started at #{start_time}"
+      
+      puts "\n!!!!! DRY RUN !!!!!\nNO DATA WILL BE IMPORTED\n" if @dry_run
+      
+      if ENV['TRUNCATE'].upcase == 'TRUE'

```

Comment:
You're going to want to give this the same treatment as `@dry_run`. Got this when I didn't include `TRUNCATE`:

```
NoMethodError: undefined method `upcase' for nil:NilClass
Did you mean?  case
/Users/chris/code/openbrewerydb/openbrewerydb-api-server/app/services/import/breweries.rb:27:in `perform'
```

Reply:
Fixed https://github.com/openbrewerydb/openbrewerydb-rails-api/pull/68/commits/9b5e26a749222b22247b172bd132448cd7aac6bf